custom_components/fire_protection_hu/sensor.py

```diff
@@ -71,14 +71,16 @@ def __init__(self, hass, name, county_id):
         self._name = name
         self._county_id = county_id
         self._state = None
-        self._fdata = [None] * 21
+        self._fdata = [None] * 20
         self._icon = DEFAULT_ICON
         self._session = async_get_clientsession(hass)
 
     @property
     def extra_state_attributes(self):
         attr = {}
         attr["provider"] = CONF_ATTRIBUTION
+        attr["county_id"] = int(self._county_id)
+        attr["data"] = self._fdata
```

Comment:
What's the point adding all/country-wide fire protection related data as attribute? It would make a lot of "noise" in the attributes.
People usually use data from their own county.
Multiple sensors can be defined if someone is interested in other county's data.